addons/api/tests/unit/serializers/host-test.js

```diff
@@ -57,6 +57,7 @@ module('Unit | Serializer | host', function (hooks) {
           address: '10.0.0.1',
           dns_names: [],
           ip_addresses: [],
+          host_set_ids: [],
```

Comment:
Nice!! testing also when a host is created without belonging to a host-set, nicely done!